API.md

```diff
@@ -52,6 +53,70 @@ Request LeadDesk to make assisted call to given number. This will also open the
 | params | <code>object</code> |  |
 | params.phoneNumber | <code>string</code> | Phone number to make the assisted call. Number needs to be in E.164 or pure local number format without any extra characters. |
 
+<a name="LeadAppMethods+contactModify"></a>
+
+### leadAppMethods.contactModify(params) ⇒ <code>Promise.&lt;void&gt;</code>
+Request LeadDesk to modify a given contact.
+
+**Kind**: instance method of [<code>LeadAppMethods</code>](#LeadAppMethods)  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| params | <code>object</code> |  |
+| params.first_name | <code>string / null</code> | First name of the contact to be modified
+| params.last_name | <code>string / null</code> | Last name of the contact to be modified
+| params.ssc | <code>string / null</code> | SSC of the contact to be modified
+| params.title | <code>string / null</code> | Title of the contact to be modified
+| params.www | <code>string / null</code> | Website of the contact to be modified
+| params.birth_year | <code>string / null</code> | Id of the contact to be modified
+| params.gender | <code>string / null</code> | Id of the contact to be modified
+| params.address.country | <code>string / null</code> | Country of the contact to be modified
+| params.address.city | <code>string / null</code> | City of the contact to be modified
+| params.address.postal_code | <code>string / null</code> | Postal Code of the contact to be modified
+| params.address.street_address | <code>string / null</code> | Street Address of the contact to be modified
+| params.email | <code>string / null</code> | Email of the contact to be modified
+| params.other_infos[].id | <code>intger</code> | Number of the other info. For example 1, 2, 3, etc it (Other info 1, Other info 2)
+| params.other_infos[].value | <code>string</code> | Value for other field 1
+
+**Example:** 
```

Comment:
Hmm, did you write these manually? This is generated by a script from the js source file "npm run apidoc".